### PR TITLE
WIP: Update core dependencies

### PR DESCRIPTION
**Pull request by nullstyle**

### CODE REVIEW

**Comment by poliha** [comment]
Looks good so far, just once concern. Won't changes here need to be propagated across(or at least tested against) js-stellar-base and/or  js-stellar-sdk?

**Comment by nullstyle** [comment]
@poliha yeah, these changes will need to happen across all of our js libraries.  Right now the plan is to update the code for all of them and get them consistent and working before we publish to npm.  Once we have the code all updated, we publish back to back.  sound good?

File: gulpfile.js [hunk 1]
```diff
@@ -37,22 +40,40 @@ gulp.task('hooks:precommit', ['build'], function() {
...
+
+  return gulp.src('src/browser.js').pipe(webpack(wconf))
+});
+
 gulp.task('build:node', ['lint:src'], function() {
     return gulp.src('src/**/*.js')
-        .pipe(plugins.babel())
+        .pipe(plugins.babel({
+          presets: ['env'],
+          plugins: [["transform-runtime", { "polyfill": false }]]
```
Comment (bartekn) [inline gulpfile.js]: Can you confirm `transform-runtime` plugin does not break React-based apps (fixed in #9)? I glanced at the [docs](https://babeljs.io/docs/plugins/transform-runtime/#technical-details) and it seems that it adds `babel-runtime` dependency that caused problems.

Comment (nullstyle) [reply gulpfile.js]: For sure, I'll check to make sure.  The project I'm working on is using react, so we should have a test case to work against.

As far as #9, the transform-runtime plugin is the replacement of the babel-runtime to solve the issue you were running into.  https://github.com/babel/babel-loader#babel-is-injecting-helpers-into-each-file-and-bloating-my-code has some info.

**Pull request closed without merging**